code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -143,7 +143,7 @@
 	var/triple_metabolism = FALSE
 	var/reduced_hunger = FALSE
 	desc = "The virus causes the host's metabolism to accelerate rapidly, making them process chemicals twice as fast,\
```

Comment:
```suggestion
	desc = "The virus causes the host's metabolism to accelerate rapidly, making them process chemicals twice as fast, \
```
Since you've deleted the space under it. This could be a single line, though, really.